examples/shader/post_processing.rs

```diff
@@ -401,7 +391,8 @@ fn update_settings(mut settings: Query<&mut PostProcessSettings>, time: Res<Time
         // Scale it to a more reasonable level
         intensity *= 0.015;
 
-        // Set the intensity. This will then be extracted to the render world and uploaded to the gpu automatically.
+        // Set the intensity.
+        // This will then be extracted to the render world and uploaded to the gpu automatically.
```

Comment:
```suggestion
        // This will then be extracted to the render world and uploaded to the gpu automatically <via the uniform component plugin thing...>.
```